NitroxClient/GameLogic/ExosuitModuleEvent.cs

```diff
@@ -212,5 +220,27 @@ public void UseTorpedo(ExosuitTorpedoArm torpedoArm, ExosuitArmAction armAction,
             }
         }
 
+        private void ToggleDrillSound(ExosuitDrillArm drillArm, bool toggled)
```

Comment:
I don't like the word `Toggle` here because when you call this method again it won't actually toggle but use the `toggled` parameter to do what you want. `Set` isn't great either but is more in line with what it does.
```suggestion
        private void SetDrillSound(ExosuitDrillArm drillArm, bool play)
```